05-lection3/02-modal/modal.css

```
@@ -0,0 +1,70 @@
+:root {
+  --main: white;
+  --primary: #4263EB;
+  --modal-title-color: #334D6E;
+  --modal-text-color: #495057;
+}
+
+body {
+  display: flex;
+  height: 100vh;
+  justify-content: center;
+  align-items: center;
+}
+
+#modal {
+  display: none;
+  position: fixed;
+  z-index: 1;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  overflow: auto;
+  background-color: rgb(0, 0, 0);
+  background-color: rgba(0, 0, 0, 0.4);
```

```suggestion
  background-color: rgba(0, 0, 0, 0.4);
```

Одна строчка перетирает другую) значит, одна из них не нужна)

---

да.. что-то невнимательно делал, забыл удалить =)